src/sentry/utils/snuba.py

```diff
@@ -222,7 +222,7 @@ def increment(
         # our requests are POST and they don't mutate, so they
         # are safe to retry. Without this, we aren't
         # actually retrying at all.
```

Comment:
This comment doesn't seem universally true anymore (some `POST` requests _do_ mutate now), and it doesn't mention anything about `DELETE`.